docs/advanced/analytics/plugins.mdx

```diff
@@ -0,0 +1,340 @@
+---
+sidebar_position: 2
```

Comment:
**nitpick:**  I think that the tracking API is more important than plugins.\
Hypothesis: an integrator or newcomer might more often want to see how to track new events than creating a new plugin

**suggestion:** swap sidebar positions